kibble/models/config.go

```diff
@@ -33,6 +33,7 @@ type Config struct {
 	DefaultTimeZone           string                    `json:"defaultTimeZone"`
 	DefaultDateFormat         string                    `json:"defaultDateFormat"`
 	DefaultTimeFormat         string                    `json:"defaultTimeFormat"`
+	RemoteTranslations 		  bool						`json:"remoteTranslations"`
```

Comment:
Too many spaces!  If you get the Go extension for VSCode then it should automatically format these things nicely.